tools/cldr-code/src/test/java/org/unicode/cldr/unittest/TestSupplementalInfo.java

```diff
@@ -1594,7 +1594,7 @@ public void TestSupplementalCurrency() {
                 // TODO 17397: remove isKnownIssue and the if around errln when the logknown issue
                 // goes away.
                 final boolean skipKnownIssue =
-                        currency.equals("ANG")
+                        currency.equals("XCG")
                                 && isoCountries.isEmpty()
```

Comment:
The preferred currency changes as of 2025-Mar-31, so updated th test skip, and also noted this in the CLDR ticket for the test skip.